src/module/linux/linux.h

```diff
@@ -12,10 +12,12 @@
     if (written < 0) {                                                                             \
       printk(KERN_WARNING "krf: snprintf formatting error\n");                                     \
     } else if (written >= KRF_NETLINK_BUF_SIZE) {                                                  \
-      printk(KERN_WARNING "krf: truncated message\n");                                             \
-      krf_netlink_broadcast(krf_log_msg_buf, KRF_NETLINK_BUF_SIZE);                                \
-    } else {                                                                                       \
-      krf_netlink_broadcast(krf_log_msg_buf, written + 1);                                         \
+      printk(KERN_WARNING "krf: truncated message\n");
+      if(krf_log_faults)                                                                           \
+        krf_netlink_broadcast(krf_log_msg_buf, KRF_NETLINK_BUF_SIZE);                              \
+    } else {
+      if(krf_log_faults)                                                                           \
+        krf_netlink_broadcast(krf_log_msg_buf, written + 1);                                       \
```

Comment:
`make fmt` for the style here, please!

Reply:
Hello, I have already done so
[make fmt](https://gitee.com/nwpu_coder/note-picture/raw/master/img/20230728235310.png)